CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.42.0
 
+* The `UseKRaft` feature gate moves to GA stage and is permanently enabled without the possibility to disable it.
+  To use the KRaft (ZooKeeper-less Apache Kafka), you still need to use the `strimzi.io/kraft: enabled` annotation on the `Kafka` custom resources.
```

Comment:
what do you think if we also add something like " ... or migrating from an existing ZooKeeper-based cluster by using strimzi.io/kraft: migration annotation"?

Reply:
Sure, that sounds reasonable.